src/content-helper/editor-sidebar/title-suggestions/component-title-suggestion.tsx

```diff
@@ -65,7 +88,15 @@ export const TitleSuggestion = (
 	// Flag if the current title has been accepted and applied to the post.
 	const titleInUse = currentPostTitle === props.title.title;
 
-	const onClickApply = async () => {
+	const onClickApply = () => {
```

Comment:
Adding a function comment would be nice, same for `onClickReplace()` and `onClickPin()`.